src/Dialog/Dialog.js

```diff
@@ -161,6 +162,9 @@ class DialogInline extends Component {
     className: PropTypes.string,
     contentClassName: PropTypes.string,
     contentStyle: PropTypes.object,
+    innerContentClassName: PropTypes.string,
+    innerContentRounded: PropTypes.bool,
```

Comment:
What about adding a `paperClassName` and `paperProps` properties instead? That's the tradeoff we us on the next branch.